regression-test/suites/dialect_compatible/presto/date.sql

```diff
@@ -0,0 +1,51 @@
+--dialect option
+set sql_dialect='presto';
+
+SELECT EXTRACT(YEAR FROM DATE '2020-05-10');
+SELECT EXTRACT(YEAR FROM DATE '1960-05-10');
+SELECT year(DATE '2020-05-10');
+SELECT year(DATE '1960-05-10');
+SELECT EXTRACT(MONTH FROM DATE '2020-05-10');
+SELECT EXTRACT(MONTH FROM DATE '1960-05-10');
+SELECT month(DATE '2020-05-10');
+SELECT month(DATE '1960-05-10');
+SELECT EXTRACT(WEEK FROM DATE '2020-05-10');
+SELECT EXTRACT(WEEK FROM DATE '1960-05-10');
+SELECT week(DATE '2020-05-10');
+SELECT week(DATE '1960-05-10');
+SELECT EXTRACT(DAY FROM DATE '2020-05-10');
+SELECT EXTRACT(DAY FROM DATE '1960-05-10');
+SELECT day(DATE '2020-05-10');
+SELECT day(DATE '1960-05-10');
+-- SELECT EXTRACT(DAY_OF_MONTH FROM DATE '2020-05-10');
+-- SELECT EXTRACT(DAY_OF_MONTH FROM DATE '1960-05-10');
+-- SELECT day_of_month(DATE '2020-05-10');
+-- SELECT day_of_month(DATE '1960-05-10');
+-- SELECT EXTRACT(DAY_OF_WEEK FROM DATE '2020-05-10');
+-- SELECT EXTRACT(DAY_OF_WEEK FROM DATE '1960-05-10');
+-- SELECT day_of_week(DATE '2020-05-10');
+-- SELECT day_of_week(DATE '1960-05-10');
+-- SELECT EXTRACT(DOW FROM DATE '2020-05-10');
+-- SELECT EXTRACT(DOW FROM DATE '1960-05-10');
+-- SELECT dow(DATE '2020-05-10');
+-- SELECT dow(DATE '1960-05-10');
+-- SELECT EXTRACT(DAY_OF_YEAR FROM DATE '2020-05-10');
+-- SELECT EXTRACT(DAY_OF_YEAR FROM DATE '1960-05-10');
+-- SELECT day_of_year(DATE '2020-05-10');
+-- SELECT day_of_year(DATE '1960-05-10');
+-- SELECT EXTRACT(DOY FROM DATE '2020-05-10');
+-- SELECT EXTRACT(DOY FROM DATE '1960-05-10');
+-- SELECT doy(DATE '2020-05-10');
+-- SELECT doy(DATE '1960-05-10');
```

Comment:
If we don't need these queries, maybe remove them would be better?

Reply:
this will be fixed in future